index.html

```diff
@@ -19,25 +19,11 @@
       content="Unofficial transcripts of No Such Thing As A Fish, a weekly podcast about interesting facts."
     />
     <meta property="og:url" content="https://transcript.fish" />
-    <meta
-      property="og:image"
-      content="https://media.transcript.fish/images/logo.jpg"
-    />
+    <meta property="og:image" content="images/logo.jpg" />
```

Comment:
This unfortunately won't work when deployed because the index page is on `https://transcript.fish` so this will try to do `https://transcript.fish/images/logo.jpg` and fail.

This works locally because the files are all actually right next to each other in the folder but for reasons too long for this comment, very little of the assets end up actually getting hosted by GitHub and live on a different domain `media.transcript.fish` which is hosted by Cloudflare.

Reply:
Removed, didn't think of that!